factorio-map-process.sh

```diff
@@ -0,0 +1,150 @@
+#!/bin/bash
+
+## Defaults
+SERVER_BASE_PATH=/srv/factorio/maps
+MAP_TILES_PATH=images
+FACTORIO_MAP_SCRIPT="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"/factoriomap.py
```

Comment:
This looks brittle. It is using `&&`, so if the `cd` fails, you won't try to print the working directory, which I guess is good. But then this var will resolve to `/factoriomap.py`, which is for sure not right.

You should set `set -euo pipefail` as the second line of this script, which will help some. See: https://vaneyckt.io/posts/safer_bash_scripts_with_set_euxo_pipefail/. These are accepted best practice.

I think the extra `cd && pwd` step isn't needed. I would go with
```suggestion
FACTORIO_MAP_SCRIPT="$(dirname "${BASH_SOURCE[0]}")/factoriomap.py"
```
I could be wrong though, I know that "finding the source directory of the script I'm running" is a very hard problem. I've struggled with it before.